<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPE</class>
 <widget class="QDialog" name="DialogPE">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>305</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>305</width>
    <height>290</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>PE basic info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButtonX" name="pushButtonDosHeader">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Dos Header</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButtonX" name="pushButtonStub">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Stub</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButtonX" name="pushButtonNtHeaders">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>NT Headers</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButtonX" name="pushButtonDirectories">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Directories</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButtonX" name="pushButtonSections">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Sections</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButtonX" name="pushButtonOverlay">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Overlay</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidgetX" name="tableWidgetBasic">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
  <customwidget>
   <class>QTableWidgetX</class>
   <extends>QTableWidget</extends>
   <header>qtablewidgetx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
